osu.Game/Overlays/BeatmapSet/BeatmapPicker.cs

```diff
@@ -68,29 +71,46 @@ public BeatmapPicker()
                             OnLostHover = () =>
                             {
                                 showBeatmap(Beatmap.Value);
-                                starRating.FadeOut(100);
+                                starRatingContainer.FadeOut(100);
                             },
                         },
                         new FillFlowContainer
                         {
                             AutoSizeAxes = Axes.Both,
                             Spacing = new Vector2(5f),
-                            Children = new[]
+                            Children = new Drawable[]
                             {
                                 version = new OsuSpriteText
                                 {
                                     Anchor = Anchor.BottomLeft,
                                     Origin = Anchor.BottomLeft,
                                     Font = OsuFont.GetFont(size: 17, weight: FontWeight.Bold)
                                 },
-                                starRating = new OsuSpriteText
+                                starRatingContainer = new FillFlowContainer
                                 {
                                     Anchor = Anchor.BottomLeft,
                                     Origin = Anchor.BottomLeft,
-                                    Font = OsuFont.GetFont(size: 11, weight: FontWeight.Bold),
-                                    Text = "Star Difficulty",
                                     Alpha = 0,
+                                    Direction = FillDirection.Horizontal,
+                                    Spacing = new Vector2(2f, 0),
                                     Margin = new MarginPadding { Bottom = 1 },
+                                    Children = new[]
+                                    {
+                                        starRatingText = new OsuSpriteText
+                                        {
+                                            Anchor = Anchor.BottomLeft,
+                                            Origin = Anchor.BottomLeft,
+                                            Font = OsuFont.GetFont(size: 11, weight: FontWeight.Bold),
+                                            Text = BeatmapsetsStrings.ShowStatsStars,
+                                        },
+                                        starRating = new OsuSpriteText
+                                        {
+                                            Anchor = Anchor.BottomLeft,
+                                            Origin = Anchor.BottomLeft,
+                                            Font = OsuFont.GetFont(size: 11, weight: FontWeight.Bold),
+                                            Text = string.Empty,
+                                        },
```

Comment:
Rationale for doing so was that the string doesn't contain any format symbols (yet? I guess) but in any case this will require updating the usages for this string on osu!-side if it ever were to change to integrate format symbols... but I agree this is hacky.

Reply:
Yeah this is fine to have, I was just remarking on it in passing. This overlay is sort of living on borrowed time anyway given the new design is just behind the corner.